docs/build/smart-contracts/samples/kaiagreeter.md

```diff
@@ -33,7 +33,7 @@ contract KaiaGreeter is Mortal {
 
 * Please visit [Kaia Plugin for Remix](https://ide.kaia.io) and create a `KaiaGreeter` contract. The complete source code was given in the above.
 * Prepare your account which will be used to deploy the contract.
-  * If you do not have an account yet, create one at [https://toolkit.kaia.io/account/accountKeyLegacy](https://toolkit.kaia.io/account/accountKeyLegacy).
+  * If you do not have an account yet, create one using [Kaia Toolkit](https://toolkit.kaia.io/account/).
   * Get some test KAIA from the faucet - [https://kairos.wallet.kaia.io/faucet](https://kairos.wallet.kaia.io/faucet)
```

Comment:
faucet link provided isn't accessible: https://kairos.wallet.kaia.io/faucet

use this instead: https://faucet.kaia.io/